### PR TITLE
shards 0.4.0 (new formula)

### PR DESCRIPTION
**Pull request by genki**
Dependency manager for the Crystal language

### CODE REVIEW

File: Library/Formula/shards.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  depends_on "manastech/crystal/crystal-lang"
+  depends_on "libyaml"
+
+  def install
+    system "make"
+    bin.install "bin/shards"
+  end
+
+  test do
+    assert(`#{bin/"shards"} --version`.chomp == "Shards 0.4.0 (2015-09-19)")
```
Comment (dunn) [inline Library/Formula/shards.rb]: This should be written as 
```ruby
assert_equal "Shards 0.4.0 (2015-09-19)", shell_output("#{bin/"shards"} --version").chomp
```

Comment (genki) [inline Library/Formula/shards.rb]: Thanks, I've fixed.

File: Library/Formula/shards.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+class Shards < Formula
+  desc "Dependency manager for the Crystal language"
+  homepage "https://github.com/ysbaddaden/shards"
+  url "https://github.com/ysbaddaden/shards/archive/v0.4.0.tar.gz"
+  sha256 "fff0f3c6562023fd279c80becf3683d2ac922ff097dda12f544f51f4b383ae5c"
+
+  depends_on "manastech/crystal/crystal-lang"
```
Comment (DomT4) [inline Library/Formula/shards.rb]: You can't have mandatory cross-tap dependencies inside core Homebrew formula, and we won't accept optional cross-tap dependencies for a tap that Homebrew doesn't have supervision over. If it needs this dependency, you'd be better hosting this in a tap.

Comment (genki) [inline Library/Formula/shards.rb]: ok. I understood.

Comment (bfontaine) [inline Library/Formula/shards.rb]: Crystal has been added in Homebrew; you can use `depends_on "crystal-lang"` now :wink:

**Comment by bfontaine** [comment]
Ping?

**Comment by bfontaine** [comment]
Closing due to no response. Feel free to open another PR addressing the comments above. Thanks!

**Pull request closed without merging**